pkg/remote/aws/error.go

```diff
@@ -0,0 +1,61 @@
+package aws
+
+import "fmt"
+
+type ListError interface {
+	error
```

Comment:
As `SupplierError` already wrap `error` is it required to compose it there too ?

Reply:
Good question I think it's not required indeed